demos/serverless/deploy.js

```diff
@@ -33,6 +33,8 @@ let mediaCaptureRegions = [
     "us-east-2",
     "us-west-1",
     "us-west-2",
+    "us-gov-west-1",
+    "us-gov-east-1",
```

Comment:
Similar question to readiness checker -- not sure that these changes are relevant to live transcription. Furthermore, is media capture supported in GovCloud?

Reply:
confirmed that media capture is not supported in govCloud regions here: https://docs.aws.amazon.com/chime-sdk/latest/dg/sdk-available-regions.html. 

will remove these lines.